tests/autorestart/test_container_autorestart.py

```diff
@@ -346,6 +346,9 @@ def test_containers_autorestart(duthosts, rand_one_dut_hostname, tbinfo):
             pytest.fail("Failed to get critical group and process lists of container '{}'".format(container_name))
 
         for critical_process in critical_process_list:
+            # Skip 'dsserve' process since it was not managed by supervisord
```

Comment:
Suggest adding a TODO comment to remove this check once it is sorted out in the image.

Reply:
Great suggestion!

Reply:
Updated!